app/helpers/bs4/bs4_header_helper.rb

```diff
@@ -11,7 +11,7 @@ def bs4_portal_type(controller) # rubocop:disable Metrics/AbcSize, Metrics/Cyclo
       if current_user.nil?
         nil
       elsif current_user.try(:has_hbx_staff_role?)
-        link_to(l10n("layout.header.role.admin"), main_app.exchanges_hbx_profiles_root_path)
+        link_to(l10n("layout.header.role.admin"), main_app.exchanges_hbx_profiles_root_path, data: { turbolinks: false })
```

Comment:
Disabling turbolinks should be gated behind the same feature flag as the new chat feature.  This is true pretty much everywhere we've decided to set turbolinks: false.